index.bs

```diff
@@ -429,6 +430,12 @@ The API applies only to {{HTMLVideoElement}} in order to start on a minimal
 viable product that has limited security issues. Later versions of this
 specification may allow PIP-ing arbitrary HTML content.
 
+## Secure Context ## {#secure-context}
+
+The API is not limited to [[SECURE-CONTEXTS]] because it exposes a feature
```

Comment:
Maybe "The API is not limited to SECURE-CONTEXTs because it is not likely to suffer from compatibility issues with existing MITM proxies, does expose the user to increased risks in case of content injection by an MITM proxy and does not expose any information about the user in case of passive monitoring of the user's network."?

Reply:
Isn't that a bit too extensive? I mean, I think the current description may benefit from more details but that sounds a bit too much. We could just mention that the feature, in addition of being built on top of features that are not limited to Secure Contexts isn't "adding any capabilities that can be a privacy or security risks".

Reply:
Why does this need to be justified at all? I'd remove this justification - it's not needed.